content/id/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -0,0 +1,1202 @@
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     1/1       Running   0          2s
+web-1     0/1       Pending   0         0s
+web-1     0/1       Pending   0         0s
+web-1     0/1       ContainerCreating   0         0s
+web-1     1/1       Running   0         34s
+```
+
+Gunakan perintah `kubectl exec` dan `kubectl run` untuk menampilkan nama hos Pod
+dan entri DNS mereka dalam klaster. Pertama-tama, tampilkan nama hos Pod:
+
+```shell
+for i in 0 1; do kubectl exec web-$i -- sh -c 'hostname'; done
+```
+```
+web-0
+web-1
+```
+selanjutnya, jalankan:
+```
+kubectl run -i --tty --image busybox:1.28 dns-test --restart=Never --rm /bin/sh
+```
+perintah itu akan menjalankan _shell_ baru.
+Di dalam _shell_ yang baru jalankan:
+```shell
+# Jalankan ini di dalam shell Container dns-test
+nslookup web-0.nginx
+```
+Keluarannya akan seperti:
+```
+Server:    10.0.0.10
+Address 1: 10.0.0.10 kube-dns.kube-system.svc.cluster.local
+
+Name:      web-0.nginx
+Address 1: 10.244.1.7
+
+nslookup web-1.nginx
+Server:    10.0.0.10
+Address 1: 10.0.0.10 kube-dns.kube-system.svc.cluster.local
+
+Name:      web-1.nginx
+Address 1: 10.244.2.8
+```
+
+(dan selanjutnya keluarlah dari _shell_ Container dengan menjalankan: `exit`)
+
+Urutan, nama hos, SRV _record_, dan nama A _record_ dari Pod tidak berubah,
+akan tetapi alamat IP yang terkait dengan Pod bisa jadi mengalami perubahan. Pada klaster
+yang digunakan dalam tutorial ini terjadi perubahan. Karena itulah mengapa sangat penting
+untuk menghindari pengaturan terhadap aplikasi lain yang terhubung ke Pod di dalam
+StatefulSet menggunakan alamat IP.
+
+Jika kamu ingin mencari dan terhubung dengan anggota aktif dari StatefulSet, kamu
+perlu melakukan kueri CNAME dari Service _headless_ (`nginx.default.svc.cluster.local`).
+SRV _record_ yang terkait dengan CNAME hanya akan memuat Pod dari StatefulSet yang
+Running dan Ready.
+
+Jika aplikasimu telah menerapkan logika koneksi yang menguji keaktifan
+(_liveness_) dan kesiapan (_readiness_), kamu dapat menggunakan SRV _record_ dari Pod (
+`web-0.nginx.default.svc.cluster.local`,
+`web-1.nginx.default.svc.cluster.local`), karena mereka tidak akan berubah, dan
+aplikasimu akan bisa menemukan alamat-alamat Pod ketika mereka mengalami peralihan
+ke Running dan Ready.
+
+### Menulis ke Penyimpanan Tetap
+
+Dapatkan PersistentVolumeClaim untuk `web-0` dan `web-1`:
+
+```shell
+kubectl get pvc -l app=nginx
+```
+Keluarannya akan seperti:
+```
+NAME        STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
+www-web-0   Bound     pvc-15c268c7-b507-11e6-932f-42010a800002   1Gi        RWO           48s
+www-web-1   Bound     pvc-15c79307-b507-11e6-932f-42010a800002   1Gi        RWO           48s
+```
+
+Pengontrol StatefulSet membuat dua
+{{< glossary_tooltip text="PersistentVolumeClaim" term_id="persistent-volume-claim" >}}
+yang terikat ke dua
+{{< glossary_tooltip text="PersistentVolume" term_id="persistent-volume" >}}.
```

Comment:
we might not have Indonesian glossary for PersistentVolumeClaim and PersistentVolume, can you help localizing these in separate PRs?

Reply:
sure...